app/src/main/java/com/kickstarter/viewmodels/ProfileViewModel.java

```diff
@@ -141,6 +148,9 @@ public ViewModel(final @NonNull Environment environment) {
       )
         .map(p -> p.first || p.second);
 
+      this.messagesButtonHidden = this.currentConfig.observable()
+        .map(config -> !config.features().getOrDefault("android_messages", false));
```

Comment:
so I think we want the default to be `true`, i.e. if we deleted the feature flag from the server we would interpret that as the feature being _on_ for everyone, not _off_.

does that make sense?